models/models_methods.go

```diff
@@ -268,6 +268,46 @@ func (config *BitriseDataModel) Validate() ([]string, error) {
 	}
 	// ---
 
+	// containers
+	for containerID, containerDef := range config.Containers {
+		if containerID == "" {
+			return nil, fmt.Errorf("service (image: %s) has empty ID defined", containerDef.Image)
+		}
+		if containerDef.Image == "" {
```

Comment:
nit: I'd also strip whitespace before checking for `""`, it's probably what the Docker API is doing anyway

Reply:
Updated in: https://github.com/bitrise-io/bitrise/pull/952/commits/34935bcbef5c72369b8ea1d9fac8a177fc6eca06